### PR TITLE
Changes Kubectl --version command's output format

### PR DESCRIPTION
**Pull request by dipankardas011**
Signed-off-by: Dipankar Das <dipsonu10@hotmail.com>

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -97,9 +97,10 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```
    Or use this for detailed view of version:
```
Comment (jihoon-seo) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:98]: Preview: https://deploy-preview-32515--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#:~:text=Some%20More%20About%20Output%20Options

IMO, the indentation of newly added text is not appropriate.
image

How about this?
```suggestion
   Or use this for detailed view of version: ([More about output options](/docs/reference/kubectl/#output-options))
```

File: content/en/docs/tasks/tools/install-kubectl-macos.md [hunk 2]
```diff
@@ -108,14 +108,16 @@ The following methods exist for installing kubectl on macOS:
...
+   ```zsh
    kubectl version --client
    ```
    Or use this for detailed view of version:
 
-   ```cmd
-   kubectl version --client --output=yaml
+   ```zsh
+   kubectl version --client --output <format> # yaml or json
+
```
Comment (jihoon-seo) [inline content/en/docs/tasks/tools/install-kubectl-macos.md:118]: Please remove unwanted newline here!

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 3]
```diff
@@ -68,9 +68,10 @@ The following methods exist for installing kubectl on Windows:
    ```
    Or use this for detailed view of version:
 
-   ```cmd
-   kubectl version --client --output=yaml     
+   ```ps
```
Comment (jihoon-seo) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:71]:
```suggestion
   ```powershell
```

image

**Review by jihoon-seo — COMMENTED** [review]
Preview: https://deploy-preview-32515--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#:~:text=Some%20More%20About%20Output%20Options

@dipankardas011 PTAL!

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 4]
```diff
@@ -95,10 +95,10 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```bash
    kubectl version --client
    ```
-   Or use this for detailed view of version:
+   Or use this for detailed view of version: [Some More About Output Options](https://kubernetes.io/docs/reference/kubectl/overview/#output-options)
```
Comment (jihoon-seo) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:98]: [This PR as-is]
image

[My suggestion]
```suggestion
   Or use this for detailed view of version ([available output options](/docs/reference/kubectl/#output-options)):
```
(Please note that I updated the link URL too.)

**Review by jihoon-seo — COMMENTED** [review]
/label tide/merge-method-squash

File: content/en/docs/tasks/tools/install-kubectl-linux.md (hunk 4 above)
Comment (dipankardas011) [reply content/en/docs/tasks/tools/install-kubectl-linux.md:98]: So adding to all the 3 👍🏼

**Comment by dipankardas011** [comment]
.

**Pull request closed without merging**